compiler/rustc_hir_analysis/src/check/wfcheck.rs

```diff
@@ -1651,6 +1651,14 @@ fn check_fn_or_method<'tcx>(
     }
 }
 
+/// The `arbitrary_self_types_pointers` feature implies `arbitrary_self_types`.
+#[derive(Clone, Copy)]
```

Comment:
I would derive at least `PartialEq` and then you can avoid a bunch of `matches!(foo, ArbitrarySelfTypesLevel::...)` elsewhere.